nixos/modules/services/networking/networkmanager.nix

```diff
@@ -229,15 +229,19 @@ in {
             source = mkOption {
               type = types.path;
               description = ''
-                A script.
+                Path to the hook script. Beware of the networkmanager dispatcher
+                service PATH (coreutils/iproute are available).
```

Comment:
First sentence is fine. Second is redundant I think (it would be rather unexpected to NOT have coreutils available). Probably can be merged with description of `type` if it turns out that `type` option isn't actually supported.

Reply:
coreutils wasn't available until this PR so I was not sure it's sthg to be assumed for nixos.

Reply:
I agree, this is more of my personal preference. I'll try to explain why I think the previous situation was an actual bug. 

When NixOS presents a `types.lines` option, it is still confusing not to have basic commands in default environment, but one can fix this with inline `${package}/bin/executable`.

When NixOS presents a `types.path` option, it looks like that file/script can be used on it's own, and most probably it doesn't contain references to `/nix/store/...` executables, but rather relies on `$PATH`.

NixOS newcomers even think that user's or root's `$PATH` inherits into service's `PATH`, but then spend half a day to fix `hostname: command not found` (happend to me :( for some other NixOS options)

----

Also, second sentence refers to "networkmanager dispatcher service PATH" which is more of implementation detail. When you read NixOS manual, you won't find anywhere that "networkmanager dispatcher service PATH", only in NixOS source code. It is correct to refer to it only If it is abstracted to an option in `networkmanager.*` namespace.

Reply:
I agree it's more of an implementation detail. I wish nixos systemd side of things were documented in nixpkgs too.